www/about_team.mako

```diff
@@ -1056,6 +1031,23 @@
             </ul>
         </div>
     </li>
+<li class="col-md-3 space-mobile">
+        <div class="thumbnail">
+            <img src="../static/img/people/davydova.jpg" alt="Katya Davydova">
+            <h3>
+                Katya Davydova
+                <small> Human Resources Intern | Community</small>
+                <noscript><span class="obfuscated-email-noscript"><small style="padding-top: 5px;">kdav<span style="display:none;">null</span>ydova@co<span style="display:none;">null</span>s.io</small></span></noscript>
+            </h3>
+            <ul class="social-icons social-icons-color">
+                <li><script type="text/javascript">document.write("<n uers=\"znvygb:xqnilqbin@pbf.vb\" ery=\"absbyybj\" qngn-bevtvany-gvgyr=\"rznvy\" pynff=\"rznvy\"></n>".replace(/[a-zA-Z]/g,function(e){return String.fromCharCode((e<="Z"?90:122)>=(e=e.charCodeAt(0)+13)?e:e-26)}));</script></li>
```

Comment:
good now.

thanks!

On Wed, May 20, 2015 at 12:15 PM, Erin Braswell <notifications@github.com>
wrote:

> In www/about_team.mako
> <https://github.com/CenterForOpenScience/centerforopenscience.org/pull/365#discussion_r30719707>
> :
>
> > @@ -1056,6 +1031,23 @@
> >              </ul>
> >          </div>
> >      </li>
> > +<li class="col-md-3 space-mobile">
> > +        <div class="thumbnail">
> > +            <img src="../static/img/people/davydova.jpg" alt="Katya Davydova">
> > +            <h3>
> > +                Katya Davydova
> > +                <small> Human Resources Intern | Community</small>
> > +                <noscript><span class="obfuscated-email-noscript"><small style="padding-top: 5px;">kdav<span style="display:none;">null</span>ydova@co<span style="display:none;">null</span>s.io</small></span></noscript>
> > +            </h3>
> > +            <ul class="social-icons social-icons-color">
> > +                <li><script type="text/javascript">document.write("<n uers=\"znvygb:xqnilqbin@pbf.vb\" ery=\"absbyybj\" qngn-bevtvany-gvgyr=\"rznvy\" pynff=\"rznvy\"></n>".replace(/[a-zA-Z]/g,function(e){return String.fromCharCode((e<="Z"?90:122)>=(e=e.charCodeAt(0)+13)?e:e-26)}));</script></li>
>
> Please remove Katya's email from here and under her name, and this should
> be good to go!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CenterForOpenScience/centerforopenscience.org/pull/365/files#r30719707>
> .
>



-- 
Betsy Ferronato
Human Resources Assistant
Center for Open Science
Betsy@cos.io